design/mattermost-support.md

```diff
@@ -16,7 +16,7 @@ Steps for adding Mattermost support:
 - Add Mattermost configurations in helm chart or config.yaml.
 - While starting controller, check in config if Mattermost support is enabled.
 - If support enabled, initialize Mattermost with values from config.
-- Start a go routinue for Mattermost.
+- Start a go routine for Mattermost.
```

Comment:
I think goroutine should be the single word
Please see: https://gobyexample.com/goroutines